tests/test_utils.py

```diff
@@ -35,6 +36,19 @@ async def test_fetch_bigquery(mock_aioresponses):
     assert result == [("row1"), ("row2")]
 
 
+@mock.patch.dict(os.environ, {"BIGQUERY_PROJECT_ID": "acme-project-id"})
+async def test_fetch_bigquery_with_specific_project(mock_aioresponses):
+    with mock.patch("telescope.utils.bigquery.Client") as mocked:
+        mocked.return_value.project = "acme-project-id"
+        mocked.return_value.query.return_value.result.return_value = [
+            ("row1"),
+            ("row2"),
+        ]
+        result = await fetch_bigquery("SELECT * FROM {__project__};")
+    mocked.return_value.query.assert_called_with("SELECT * FROM acme-project-id;")
+    assert result == [("row1"), ("row2")]
```

Comment:
The mock and this assertion about `result.return_value` is not really necessary for the scope of this test

Reply:
thanks! I've changed tests in the last commit, this is outdated now..